### PR TITLE
[TEST] Remove deprecated FunSuite

### PR DESCRIPTION
**Pull request by pan3793**
### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: scalastyle-config.xml [hunk 1]
```diff
@@ -136,7 +136,7 @@ This file is divided into 3 sections:
   <check level="error" class="org.scalastyle.scalariform.NotImplementedErrorUsage" enabled="true"/>
 
   <check customId="funsuite" level="error" class="org.scalastyle.scalariform.TokenChecker" enabled="true">
-    <parameters><parameter name="regex">^FunSuite[A-Za-z]*$</parameter></parameters>
+    <parameters><parameter name="regex">^AnyFunSuite[A-Za-z]*$</parameter></parameters>
```
Comment (yaooqinn) [inline scalastyle-config.xml:139]: so there will be no check for `FunSuite`?

Comment (pan3793) [reply scalastyle-config.xml:139]: `FunSuite` is deprecated in scalatest 3.1, and replaced by `AnyFunSuite`

Comment (yaooqinn) [reply scalastyle-config.xml:139]: I know, it's deprecated but not removed so we still need this checker?

Comment (pan3793) [reply scalastyle-config.xml:145]: updated.

**Review by yaooqinn — APPROVED** [review]

**Review by turboFei — APPROVED** [review]

**Comment by turboFei** [comment]
thanks, merging to master

**Pull request closed without merging**